### ISSUE TITLE
add units to min / max age

### ISSUE THREAD

**Issue by zednis** [tetherless-world/dtdi-mineral-observation-map#11]
Min and max age values are displayed as just integers.  

![screen shot 2016-01-27 at 10 04 36 am](https://cloud.githubusercontent.com/assets/1057295/12621368/8565a0ae-c4dd-11e5-8bda-70626f6c6be5.png)

I believe these values are in millions of years.  I think we should update the UI to show the unit in both the search result record.

I think we should add the units after the displayed integer in the mineral observation record.

What do you think @xgmachina ?

Also, I am curious if there is other information relevant to the estimated min and max age that we should present.  Is there a confidence value associated with the min and max age?  How is the value determined and is there expected to be a normal probability distribution for the age between the min and max?

If we know this information and it is consistent across the mineral observation records I think we should update the UI in some way to make this information accessible to users.  We should also consider updates to the JSON and eventually RDF encodings to express that knowledge.

**Comment by zhongh** [comment]
So I had an discussion with Marshall on this but we haven't had a conclusion about it yet. There are different labels for the estimated min/max ages, meaning some of them is by direct measuring, some by estimation based on nearby measures, and so and so, all together there are 4-5 different way of obtaining this estimation. But it doesn't provide quantitative information on confidence of accuracy. What do you think? @xgmachina 

By the way, talking about adding unit to min/max ages, should it be like "15,600,000 years" or "15.6 million years"?

**Comment by zednis** [comment]
No need to add unnecessary significant digits and I think it would be ok to use the unit label, "Ma", the community is familiar with.  We probably want to have a key at the bottom of the page defining "Ma" for users not familiar with the label.  I think we should also have a hover-over on the unit label to provide the full description

As discussed today on the telecon I think we should grab the information on how the age was determined and show it in the UI.  I think a superscript using the same letter as in RRUFF should be good for a first pass.  We will want to add a definition for all the superscript values in the key at the bottom (or top?) of the page.  We should also have a hover-over on the superscript that provides a definition.

So I think "15.6 Ma<sup>G</sup>" should be good with text hover-over definitions and a key at the bottom (or top) of the page with definitions.

@kolans what do you think?

**Comment by kolans** [comment]
@zednis That sounds perfect. Like you said, we should make sure to provide legends for all the abbreviations we use. I think that superscript works well, too.

**Comment by zhongh** [comment]
@kolans @zednis No problem I will put in in!

**Comment by zednis** [comment]
Another note on min/max age.  If we want to use the range facet on age values we need to store the min/max age values as numbers instead of strings.

A range facet for min age would look something like this:

```
{'type' : 'range', 'field': 'locality.min-age', 'display': 'Min Age', 'range': [{'to': 100, 'display': '0 - 100 Ma'}, {'from': 100, 'to': 200, 'display': '100 - 200 Ma'}, {'from': 200, 'display': 'greater than 200 Ma'}], 'open': 'true'},
```

but will not work with the current elasticsearch JSON where age values are stored as strings.

**Comment by zednis** [comment]
I will actually create a new ticket for updating the age values to be numbers.

**Comment by zednis** [comment]
proposal: add a age-unit property under mineral-specimen with type string.

**Comment by zednis** [comment]
Hmm, this should have been automatically closed by #29

**Comment by zhongh** [comment]
@zednis I found that it seems that if we want to automatically close an issue from a pull request, we should use "fixed" instead of "fixes".
